app/queries/keywords_query.rb

```diff
@@ -2,7 +2,7 @@
 
 class KeywordsQuery
   def initialize(user)
-    @keywords = user.keywords
+    @keywords = user.keywords.select(Keyword.column_names - ['html'])
```

Comment:
Rails 6 introduced `excluding` method, how about to use it? I think it is a bit more redable
```suggestion
    @keywords = user.keywords.select(Keyword.column_names.excluding("name"))
```

Reply:
Sweeeeeeet! Thanks!

Reply:
(total agree for the clarity to read the code!)